mycore-migration-2015/src/main/java/org/mycore/migration2015/cli/MCRMigrationCommands.java

```diff
@@ -1,48 +1,62 @@
 package org.mycore.migration2015.cli;
```

Comment:
the fix goes into the LTS but we can rename mycore-migration-2015 to mycore-migration in master with another ticket